uiowa/uiowa
# Admissions - AoS Search Filtering View

Resolves https://github.com/uiowa/uiowa/issues/3203

## To Test

- pull and sync admissions
- give at least one existing area of study a pretty title. remember it.
- go to http://admissions.local.drupal.uiowa.edu/areas-of-study
- search should filter on contains area of study title or pretty title. remember your pretty title?
- academic group, college, program type should act like normal views filters if the content exist.
- degree/type column should render similar to "BA, BS, Minor, Certificate, Online Option" if the content exists, otherwise the string reflects that and their are no trailing commas.
- i am sure i am forgetting something, but it is probably covered by this awesome PR.
